IDL/sequences.idl

```diff
@@ -0,0 +1,84 @@
+enum MyEnum
+{
+    ENUM_VALUE_1,
+    ENUM_VALUE_2,
+    ENUM_VALUE_3
+};
+
+@bit_bound(8)
+bitmask MyBitMask

```

Comment:
I join these helper types (equal in several idl files) in a outer idl file `helpers/basic_inner_types.idl`

Reply:
created in https://github.com/eProsima/dds-types-test/pull/1/commits/7807affbc3bdb9302cd109f1e64c8a0bf837d7c8